components/event-bus/cmd/mesh-namespaces-migration/serviceinstance.go

```diff
@@ -13,17 +13,23 @@ import (
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes/scheme"
 
+	appconnectorv1alpha1 "github.com/kyma-project/kyma/components/application-operator/pkg/apis/applicationconnector/v1alpha1"
 	kymaeventingclientset "github.com/kyma-project/kyma/components/event-bus/client/generated/clientset/internalclientset"
 )
 
 const (
-	appBrokerServiceClass = "application-broker"
-	serviceInstanceKind   = "ServiceInstance"
+	appBrokerServiceClass  = "application-broker"
+	serviceInstanceKind    = "ServiceInstance"
+	eventsServiceEntryType = "Events"
 )
 
-type serviceInstanceList []servicecatalogv1beta1.ServiceInstance
+type servicesInstanceList []servicecatalogv1beta1.ServiceInstance
```

Comment:
Will `serviceInstancesList` make sense as it is close to `name: serviceinstances.servicecatalog.k8s.io`?

Reply:
Done 852acf5d4c513f181759809f5843e57df07e2110